app/policies/upcoming/manage/facility_group_policy.rb

```diff
@@ -0,0 +1,27 @@
+class Upcoming::Manage::FacilityGroupPolicy < Upcoming::ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def allowed?
+    return true if user.super_admin?
+    user.accesses.admin.where(resource: resolve_record(record, FacilityGroup)).exists?
+  end
+
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    def resolve
```

Comment:
This again, is a pattern repeated throughout all policies which just calls into the `Access` methods